core/storage/suggestion/gae_models_test.py

```diff
@@ -6028,7 +6028,18 @@ class SortChoicesUnitTest(test_utils.GenericTestBase):
     """
 
     def test_enum_dictionary_match(self) -> None:
+        enum_values = []
+        dict_values = []
         for enum_member in suggestion_models.SortChoices:
+            enum_values.append(enum_member.value)
             self.assertTrue(
```

Comment:
You don't need this any more, right?

Reply:
Yeah. it doesn't make sense to check this now.